packages/react-devtools-extensions/src/main.js

```diff
@@ -107,6 +108,10 @@ function createPanelIfReactLoaded() {
 
       const tabId = chrome.devtools.inspectedWindow.tabId;
 
+      registerEventLogger((_event: LogEvent) => {
+        // TODO: hook up event logging
```

Comment:
i think the most straightforward next step here would be to log these events to an internal endpoint when the feature flag is enabled, and using a secret to provide the endpoint. 

other alternatives could include injecting code code in react during the build step, which would be accessed here, but i think this approach can give us a bit more flexibility. happy to discuss more or put up a specific proposal for  what the next steps could be here

Reply:
Can we setup this endpoint to make sure it's feasible? (I Have no experience doing this. Maybe you do?)